templates/advancement/advancement-migration-dialog.hbs

```diff
@@ -1,25 +1,32 @@
-<div class="advancement">
-    <p>{{localize "DND5E.AdvancementMigrationHint" name=item.name}}</p>
-    <ol class="items-list">
-        <li class="items-header flexrow">
-            <h3 class="item-name flexrow">{{localize "DND5E.AdvancementTitle"}}</h3>
-            <div class="item-controls flexrow">{{localize "DND5E.Add"}}</div>
-        </li>
+<fieldset class="card items">
+    <legend>{{ localize "DND5E.ADVANCEMENT.Migration.Selection" }}</legend>
+    <div class="form-group">
+        <p class="hint">{{ localize "DND5E.ADVANCEMENT.Migration.Hint" name=item.name }}</p>
+    </div>
```

Comment:
This would look a bit nicer with some extra padding. Maybe add a `.note` or `.hint` class to the `.form-group` and give it `padding: 0 8px 8px;`.

![image](https://github.com/user-attachments/assets/00368eca-2f4c-49e1-ac28-d54021fe804a)